model/postgres/postgres_server.rb

```diff
@@ -172,4 +172,8 @@ def lsn2int(lsn)
   def lsn_diff(lsn1, lsn2)
     lsn2int(lsn1) - lsn2int(lsn2)
   end
+
+  def run_query(query)
+    vm.sshable.cmd("psql -U postgres -t --csv", stdin: query).chomp
```

Comment:
we don't have `statement_timeout` configured. I am worried these statements may hang around in case of a buggy code in future. Should we put a timeout here?

Reply:
Makes sense. Added it.